ecosystem/platform/server/app/views/static_page/incentivized_testnet.html.erb

```diff
@@ -0,0 +1,80 @@
+<% content_for(:page_title, 'Incentivized Testnet - Ensuring a production-grade Aptos mainnet.') %>
+<% content_for(:page_description, 'The Aptos ecosystem is rewarding node operators, developers, content creators, project leaders, and others who help us prepare to launch a secure, reliable & decentralized network.') %>
+
+  <div class="bg-neutral-900 text-neutral-50 h-full">
+    <div class="max-w-screen-2xl mx-auto px-6 pb-24">
+      <section class="py-12 lg:py-32">
+        <h3 class="text-base uppercase text-teal-400 mb-4 font-mono">Community</h3>
+        <div class="grid grid-cols-1 md:grid-cols-2 gap-12 xl:gap-20">
+          <h1 class="font-display text-5xl lg:text-6xl xl:text-6xl text-neutral-100">
+            Ensuring a production-grade Aptos mainnet.
+          </h1>
+          <div class="flex items-center">
+            <div>
+              <p class="text-xl xl:text-2xl text-neutral-400 xl:leading-normal">
+                The Aptos ecosystem is rewarding node operators, developers, content creators, project leaders, and others who help us prepare to launch a secure, reliable & decentralized network.
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="pt-12 pb-20 lg:grid grid-cols-12">
+        <div class="lg:col-start-2 lg:col-span-10 flex align-center items-center gap-2">
+          <a href="/it1" class="flex flex-col justify-center relative group" title="View Incentivized Testnet 1 Results">
```

Comment:
Can you change these URLs to `<%= it1_path =>`, `<%= it2_path =>`, etc. This way we get errors in development if the routes don't exist, avoid typos, etc.